serving/installing-logging-metrics-traces.md

```diff
@@ -1,14 +1,14 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups: 
-One that uses Elasticsearch, Kibana, Prometheus and Grafana and 
-another that uses Stackdriver, Prometheus and Grafana. See below 
-for installation instructions for these two setups. You can install 
+Knative Serving offers two different monitoring setups:
+One that uses Elasticsearch, Kibana, Prometheus and Grafana and
+another that uses Stackdriver, Prometheus and Grafana. See below
+for installation instructions for these two setups. You can install
 only one of these two setups and side-by-side installation of these two are not supported.
 
 ## Elasticsearch, Kibana, Prometheus & Grafana Setup
```

Comment:
Under the stackdriver section, there is an option to configure the plugin. Should this Elasticseach section also mention a "configuring" option and point to "/serving/setting-up-a-logging-plugin.md" ?

Reply:
Added.